install/disconnected_install.adoc

```diff
@@ -539,218 +528,114 @@ $ systemctl enable httpd
 $ systemctl start httpd
 ----
 
-[[disconnected-openshift-systems]]
-== {product-title} Systems
+[[disconnected-standalone_registry]]
+== Populate the registry
 
-[[disconnected-building-your-hosts]]
-=== Building Your Hosts
+From within your disconnected environment, tag and push the images to your
+internal registry:
 
-At this point you can perform the initial creation of the hosts that will be
-part of the {product-title} environment. It is recommended to use the latest version
-of RHEL 7 and to perform a minimal installation. You will also
-want to pay attention to the other
-xref:prerequisites.adoc#install-config-install-prerequisites[{product-title}-specific
-prerequisites].
+[IMPORTANT]
+====
+The following steps are a generic guide to loading the images into a registry.
+You might need to take more or different actions to load the images.
+====
 
-Once the hosts are initially built, the repositories can be set up.
+. Before you push the images into the registry, re-tag each image. For example,
+to tag the etcd image:
++
+[source, bash]
+----
+$ docker tag registry.access.redhat.com/rhel7/etcd:3.2.22 registry.example.com/rhel7/etcd:3.2.22
+----
 
-[[disconnected-connecting-repos]]
-=== Connecting the Repositories
+. Push each image into the registry. For example, to push the etcd image:
++
+[source, bash]
+----
+$ docker push registry.example.com/rhel7/etcd:3.2.22
+----
+
+. <add commands to tag and push images from the other sections>
```

Comment:
When pushing image, we should pushing two tag for one image, e.g: ose-pod:v3.10.34 and ose-pod:v3.10, that would be more safe, at least for each release, RCM would add multiple tag for one image. Hope we did not lost https://github.com/openshift/openshift-docs/pull/11630 in this refactor.